source/common/modules/markdown-editor/hooks/note-preview.ts

```diff
@@ -101,9 +104,31 @@ function getPreviewElement (metadata: [string, string, number, number], linkCont
   openButton.setAttribute('id', 'open-note')
   openButton.textContent = 'Open'
   openButton.addEventListener('click', openFunc)
-
   actions.appendChild(openButton)
 
+  // Only if preference "Avoid New Tabs" is set, 
+  // offer an additional button on preview tooltip
+  // to open the file in a new tab
+  if (Boolean(global.config.get('system.avoidNewTabs'))) {
+    const openFuncNewTab = function (): void {
+      ipcRenderer.invoke('application', {
+        command: 'force-open',
+        payload: {
+                  linkContents: linkContents,
+                  newTab: true
+          }
+      })
+        .catch(err => console.error(err))
+    }
+
+    const openButtonNT = document.createElement('button')
+    openButtonNT.setAttribute('id', 'open-note-new-tab')
+    openButtonNT.textContent = 'Open in new Tab'
+    openButtonNT.addEventListener('click', openFuncNewTab)
+    openButtonNT.style.cssText += ';margin-left :10px;'
```

Comment:
Rather than using `cssText` please use the correct style itself:

`openButtonNT.style.marginLeft = '10px'`